src/include/daos_task.h

```diff
@@ -69,6 +69,7 @@ typedef enum {
 	DAOS_OPC_POOL_DEL_ATTR,
 	DAOS_OPC_POOL_STOP_SVC,
 	DAOS_OPC_POOL_LIST_CONT,
+	DAOS_OPC_POOL_TGT_INFO,
```

Comment:
Instead of adding a new OPC, you should modify the existing DAOS_OPC_POOL_QUERY_TARGET (Or just reuse it)